moped-editor/src/utils/activityLogFormatters/mopedProjectActivity.js

```diff
@@ -46,7 +46,7 @@ export const formatProjectActivity = (change, lookupList) => {
             style: null,
           },
           {
-            text: entryMap.fields[changedField].label,
+            text: entryMap.fields[changedField]?.label,
```

Comment:
this way it will just say "undefined" in the future, instead of breaking

we could also come up with a generic fallback like "a field" but i thought undefined was okay for now.

Reply:
Agreed - setting a default would be awesome but sounds like a future thing to me too 👍

Reply:
👍 i will update https://github.com/cityofaustin/atd-data-tech/issues/11105 for this task

Reply:
yes!! still relevant. ill work on it with my activity log issues